pages/boldgrid-inspirations.php

```diff
@@ -331,7 +331,7 @@
 			</div>
 			<div class="content-menu-section">
 				<div class="feature-filter"><?php echo $lang['AddFunctionality']; ?></div>
-				<div class="feature-option">
+				<div class="feature-option" id="feature_option_blog" data-shown-pointer=false>
```

Comment:
Is `data-shown-pointer` needed? I don't see any changes in the code that would look at that attribute.